docs/asset-store/10-01-performance.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Performance
```

Comment:
user can learn from this document only the default performance, no info about how to tweak it